syntax = "proto3";

package user;

service UserService {
    rpc GetUserById(UserIdRequest) returns (UserResponse);
    rpc CreateUser(CreateUserRequest) returns (UserResponse);
}

message UserIdRequest {
    int32 id = 1;
}

message CreateUserRequest {
    string email = 1;
    string full_name = 2;
}

message UserResponse {
    int32 id =1;
    string email = 2;
    string full_name = 3;
}